### PR TITLE
Frame Rate Mode and Frame Rate API description update

### PR DESCRIPTION
**Pull request by krishna-1809**
Update the description below APIs:

dsSetFRFMode()
frame rate mode :   0 for auto framerate mode disabled, 1 for auto framerate mode enabled (must be one of the following: 0, 1)

dsSetDisplayframerate(): 
"3840x2160px60"

### CODE REVIEW

File: include/dsVideoDevice.h [hunk 1]
```diff
@@ -307,8 +307,8 @@ dsError_t dsForceDisableHDRSupport(intptr_t handle, bool disable);
  * For source devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
  *
  * @param[in] handle    - The handle returned from the dsGetVideoDevice() function
- * @param[in] frfmode   - integer with corresponding Framerate value. 
- *                               Please refer ::dsVideoFrameRate_t for max and min framerate.
+ * @param[in] frfmode   - integer value to enable/disable Auto Framerate mode.
```
Comment (hari22yuva) [inline include/dsVideoDevice.h:310]: is it possible to use bool rather integer?

File: include/dsVideoDevice.h [hunk 2]
```diff
@@ -388,9 +387,8 @@ dsError_t dsGetCurrentDisplayframerate(intptr_t handle, char *framerate);
  * For source devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
  *
  * @param[in] handle    - The handle returned from the dsGetVideoDevice() function
- * @param[in] framerate - Framerate value to be set frame will be represented in FPS. 
- *                        Please refer ::dsVideoFrameRate_t for  max and min framerate.
- *                        Expects the value as a string(eg:"60").
+ * @param[in] framerate - Framerate value to be set.
```
Comment (hari22yuva) [inline include/dsVideoDevice.h:390]: is this frame rate values right?
please check this
https://github.com/rdkcentral/rdk-halif-device_settings/blob/feature/issues72_FrameRate_Mode_desc_update/include/dsAVDTypes.h#L508

**Review by hari22yuva — COMMENTED** [review]
please address the review comments

File: include/dsVideoDevice.h (hunk 1 above)
Comment (krishna-1809) [reply include/dsVideoDevice.h:310]: It sets the westeros command.
https://gerrit.teamccp.com/plugins/gitiles/rdk/components/generic/devicesettings/soc/amlogic/common/+/24Q3_sprint/dsVideoDevice.c#452

Better to use int

File: include/dsVideoDevice.h (hunk 2 above)
Comment (krishna-1809) [reply include/dsVideoDevice.h:390]: supported framerates to be updated in description
**ex: 1920x1080x24**

**Review by ddevadas53276 — APPROVED** [review]

**Review by bijasbabu — APPROVED** [review]
Ship it

**Review by hari22yuva — APPROVED** [review]
Approve

**Pull request merged**